internal/app/components/courses/courseRecords_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 )
 
-func TestCourseRecords_ToCourseIdMap(t *testing.T) {
+func TestCourseRecord_ToCourseIdMap(t *testing.T) {
```

Comment:
why are we changing the name of this func?

Reply:
consistency lmao: `Test(struct name)_(func name)`

Reply:
oh shoot i goofed

Reply:
LOL all good